cli/cmd/create_project.go

```diff
@@ -190,13 +190,19 @@ func getYamlFromURL(arg string) (string, error) {
 	if err != nil {
 		return "", err
 	}
+	createProjectParams.Content = string(body)
 	return string(body), nil
 }
 
 func getYamlFromFile(arg string) (string, error) {
 	if _, err := os.Stat(keptnutils.ExpandTilde(arg)); os.IsNotExist(err) {
 		return "", fmt.Errorf("Cannot find file %s", arg)
 	}
+	fileContent, err := file.ReadFile(arg)
+	if err != nil {
+		return "", err
+	}
+	createProjectParams.Content = fileContent
 	return file.ReadFile(arg)
```

Comment:
Since you introduced reading the file in line 201, there is no need to read it again at this place.
```suggestion
	return fileContent, nil
```

Reply:
Yeah, my bad will remove this.